File: Questions_in_jS_Console_Based/q10.js
//Find Cirumference area of sphere

let r = parseFloat(prompt('Enter the Radius:', 0));

// const cirumference = Math.round(4 * Math.PI * r * r);
const cirumference = Math.round(4 * Math.PI * Math.pow(r, 2));

const area = Math.round(2 * Math.PI * r);

alert(cirumference);
alert(area);

File: UnDirectional_Folders/Day1_Java/q10.java
import java.util.Scanner;

public class q10 {
    public static void main(String[] args) {
        System.out.println("Enter the radius of Sphere:");

        Scanner sc = new Scanner(System.in);
        double r = sc.nextDouble();

        System.out.println("Area:" + 4 * Math.PI * r * r);
        sc.close();
    }
}
